src/main/java/uk/gov/hmcts/reform/sscs/service/UserDetailsService.java

```diff
@@ -25,6 +30,26 @@ public String buildLoggedInUserSurname(String userAuthorisation) {
         return userDetails.getSurname().orElse("");
     }
 
+    public String getUserRole(String userAuthorisation) {
+        List<String> users = getUserInfo(userAuthorisation).getRoles();
+
+        for (UserRole userRole : UserRole.values()) {
```

Comment:
i think you can use streams here. filter by what you have in if statement and orElse null